scripts/build_static_site.sh

```diff
@@ -104,31 +103,25 @@ build() {
     export SOURCECRED_DIRECTORY="${sourcecred_data}"
 
     if [ "${#projects[@]}" -ne 0 ]; then
-        local weightsStr=""
-        if [ -n "${weights}" ]; then
-            weightsStr="--weights ${weights}"
-        fi
         for project in "${projects[@]}"; do
             NODE_PATH="/code/node_modules${NODE_PATH:+:${NODE_PATH}}" \
-                node /code/bin/sourcecred.js load "${project}" $weightsStr
+                node /code/bin/sourcecred.js load "${project}" \
+                ${weights:+--weights "${weights}"}
         done
     fi
 
     if [ "${#project_files[@]}" -ne 0 ]; then
-        local weightsStr=""
-        if [ -n "${weights}" ]; then
-            weightsStr="--weights ${weights}"
-        fi
         for project_file in "${project_files[@]}"; do
             NODE_PATH="/code/node_modules${NODE_PATH:+:${NODE_PATH}}" \
-                node /code/bin/sourcecred.js load --project "${project_file}" $weightsStr
+                node /code/bin/sourcecred.js load --project "${project_file}" \
+                ${weights:+--weights "${weights}"}
```

Comment:
@wchargin is something happening so that the project file isn't actually honored? Based on [combined.json](https://github.com/vsoch/sourcecred-cred/blob/master/projects/combined.json), shouldn't it be the case that we don't run sourcecred over every repository in the org?